lib/index.js

```diff
@@ -15,7 +15,7 @@ const testGeneratorNames = Object.keys(testGenerators);
 
 const FACTORY_METHOD_USED = Symbol('create() factory method was used');
 
-function resolveInputDirectory(inputNode) {
```

Comment:
Let’s revert this change. Having a module scope function is pretty idiomatic IMHO.